controllers/apply/lib/notices.js

```diff
@@ -47,6 +47,15 @@ module.exports = {
                     un newydd sy'n canolbwyntio ar COVID-19 yn lle.`,
                 }),
             });
+
+            if (enableGovCOVIDUpdates) {
+                notices.push({
+                    title: oneLine`We've also changed our eligibility
```

Comment:
No translation for this, it's only shown in England. Took the same approach as the England only terms for now.